x11-libs/fltk/fltk-1.3~git.recipe

```diff
@@ -101,19 +104,16 @@ BUILD()
 
 INSTALL()
 {
-	mkdir -p $docDir
 	make install
 
-	fixPkgconfig
-
 	prepareInstalledDevelLibs \
-		libfltk
-
-	mv -f $libDir/*.a $developLibDir/
+		libfltk libfltk_forms libfltk_images
+	fixPkgconfig
 
 	# TODO: fix configure.ac
-	mkdir -p $developDocDir
-	mv -f $dataDir/doc/fltk/* $developDocDir/
+	mkdir -p $docDir
+	mv -f $dataDir/doc/fltk/* $docDir/
+	rm -rf $dataDir/doc
```

Comment:
Still left with an empty $docDir/packages in the base package here (that has been moved to the devel package)

Reply:
Yeah I tried to clean those up but…

Reply:
fixed :)